src/array/cuda/rowwise_sampling.cu

```diff
@@ -22,6 +22,7 @@ namespace impl {
 namespace {
 
 constexpr int WARP_SIZE = 32;
```

Comment:
is `WARP_SIZE` still used?

Reply:
> How much does the launch dimension change affect the performance when you enable it on top of the random number generator change?

If not consider the change of random number generator, the change of launch dimension can significantly reduce the latency of model which has super nodes, eg: for RGCN, the latency reduces from 4ms to 1.5ms.
Combine these two changes, we can see significant change on three models.